pkg/cmd/build/view.go

```diff
@@ -57,8 +56,14 @@ func NewCmdBuildView(f *factory.Factory) *cobra.Command {
 				return fmt.Errorf("could not resolve a build")
 			}
 
+			if web {
+				buildUrl := fmt.Sprintf("https://buildkite.com/%s/%s/builds/%s", bld.Organization, bld.Pipeline, args[0])
+				fmt.Printf("Opening %s in your browser\n\n", buildUrl)
```

Comment:
I think a single newline would be better? Seeing 2 in the output when I run it makes me think something is missing
```suggestion
				fmt.Printf("Opening %s in your browser\n", buildUrl)
```